raiden/utils/cli.py

```diff
@@ -423,13 +425,27 @@ def apply_config_file(
             cli_params[config_name_int] = config_value
 
 
-def get_matrix_servers(url: str) -> List[str]:
-    """Fetch a list of matrix servers from a text url
+def get_matrix_servers(
+    url: str, server_list_type: ServerListType = ServerListType.ACTIVE_SERVERS
+) -> List[str]:
+    """Fetch a list of matrix servers from a URL
 
-    '-' prefixes (YAML list) are cleaned. Comment lines /^\\s*#/ are ignored
+    The URL is expected to point to a JSON document of the following format::
 
-    url: url of a text file
-    returns: list of urls, default schema is https
+        {
+            "active_servers": [
+                "url1",
+                "url2",
+                ...
+            ],
+            "all_servers": [
+                "url1",
+                "url2",
+                ...
+            ]
+        }
+
+    Which of the two lists is returned is controlled by the ``server_list_type`` argument.
```

Comment:
We could also just return all and let the caller choose by indexing the returned dict.

Reply:
It's not. I just like to thing aloud when reviewing PRs.